Home.md

```diff
@@ -1,40 +1,51 @@
 
-d:swarm (mainly) takes care of the **improvement** of the data quality facilitating the de-duplication, merging and [FRBRization](http://en.wikipedia.org/wiki/Functional_Requirements_for_Bibliographic_Records) of bibliographic data, their semantic enrichment and publication as LOD.
+[[http://www.dswarm.org/wp-content/uploads/2015/04/dswarm-demo_2015-04-14.png]]
 
-d:swarm channels all **institutional data flows** and is able to create a highly customized master record for every individual resource as a [Single Version of the Truth](http://en.wikipedia.org/wiki/Single_version_of_the_truth) (SVOT).
+Finally, d:swarm is an **ETL tool with a GUI for non-programmers**. Librarians do not need to write scripts, but can create complex transformations by Drag and Drop from a functions library and configuring them in the d:swarm BackOffice. Following the concept of [community sharing](http://en.wikipedia.org/wiki/Sharing), transformations, mappings and almost any other artifact in d:swarm that could be helpful to others is designed for **reuse**.
 
-d:swarm is a web application that runs in all modern web browsers. The current release of our web application is available at [http://demo.dswarm.org](http://demo.dswarm.org). If you want to participate in the tests, [drop us a note](mailto:team@dswarm.org), and we will gladly add you to the group of testers. We are looking forward to your feedback, your ideas, your opinion and your contribution at our [mailing list](https://groups.google.com/forum/#!forum/dswarm) or [issue tracker](https://jira.slub-dresden.de).
+d:swarm is realized as a web application that runs in all modern web browsers. The current release of our web application is available at [http://demo.dswarm.org](http://demo.dswarm.org). If you want to participate in the tests, [drop us a note](mailto:team@dswarm.org), and we will gladly add you to the group of testers. We are looking forward to your feedback, your ideas, your opinion and your contribution at our [mailing list](https://groups.google.com/forum/#!forum/dswarm) or [issue tracker](https://jira.slub-dresden.de).
 
-## What can you do with d:swarm?
+## What is the d:swarm idea?
 
-As a base, [this](http://prezi.com/0zh_ypsezu6i/) and [this](http://www.slideshare.net/JensMittelbach/dswarm-a-library-data-management-platform-based-on-a-linked-open-data-approach) presentations summarize the motivation and goals on an abstract level. Furthermore, our [user guide](https://github.com/dswarm/dswarm-documentation/wiki/Overview) provides a brief manual of how to utilise the [d:swarm Backoffice web application](http://demo.dswarm.org).
+Start watching these two ([#1](http://prezi.com/0zh_ypsezu6i/), [#2](http://www.slideshare.net/JensMittelbach/dswarm-a-library-data-management-platform-based-on-a-linked-open-data-approach)) presentations, which summarize the motivation and goals on an abstract level.

```

Comment:
maybe a link to more recent presentation such as https://docs.google.com/presentation/d/1Aa_JM8YjSOF3A4LPm0ZuoFWzu5h-NN7VtygAE0WG97Q/pub?start=false&loop=false&delayms=60000 ?